packages/application-extension/src/index.ts

```diff
@@ -264,12 +264,14 @@ const pages: JupyterFrontEndPlugin<void> = {
       }
     });
 
-    app.commands.addCommand(CommandIDs.openTree, {
-      label: trans.__('Open Files'),
-      execute: () => {
-        window.open(`${baseUrl}tree`);
-      }
-    });
+    if (PageConfig.getOption('notebookPage') !== 'tree') {
```

Comment:
Wondering if we need this check, or simply let folks open the file browser again if they want?

Reply:
In fact the `filebrowser-extension` already adds an entry in menu View:
https://github.com/jupyterlab/jupyterlab/blob/fc3f6ab716ef82944ab6c4e5d1a06418daad49dc/packages/filebrowser-extension/schema/browser.json#L24-L29
That would be a duplicated entry, one to switch to file browser tab, and one to open a new page.
This is also why I renamed the *Open Files* entry to *File Browser*, for consistency with the JupyterLab one.

Reply:
Thanks, I didn't know about that one.
I just realized that clicking on the same menu entry `File Browser` when the file browser is activated try to call `filebrowser:hide`, which is not currently defined in notebook. This raises an error in console.
Do you think we should defined the command `filebrowser:hide` without any action, to avoid the error ?

Reply:
Does it? I just tried with this menu entry on Binder but it does not seem to be printing anything:

![image](https://user-images.githubusercontent.com/591645/197409901-6d4c4974-6978-4366-a562-9b67cabe8d44.png)

Reply:
On binder too I have this error message in the console, when the file browser is already activated before clicking on the link:

<img src="https://user-images.githubusercontent.com/32258950/197467866-6315a657-5c8f-4ed6-a8b1-fced113d86c3.png" width="500">

The command added by JupyterLab is `filebrowser:toggle-main`, which calls `filebrowser:activate` or `filebrowser:hide-main` depending on the `isHidden` status. Maybe it is not triggered every time for some reason.

BTW I changed the way this command is defined (using `isVisible`) in https://github.com/jupyter/notebook/pull/6588/commits/8559a4df20702e7a784f3f5aae4bb2d8f37193a0

Reply:
> Do you think we should defined the command `filebrowser:hide` without any action, to avoid the error ?

We could also make the file browser tab 'closeable' and define properly the command `filebrowser:hide`.
Not sure this is expected for *Jupyter Notebook*, that may change a bit the philosophy.

Reply:
You are right.
It can be solved by fixing it in JupyterLab, as explain below: suppress the `filebrowser:toggle-main` entry from menu and create one only for `filebrowser:activate`.